cartography/cli.py

```diff
@@ -112,6 +112,24 @@ def _build_parser(self):
                 'respects the AWS CLI/SDK environment variables and does not override them.'
             ),
         )
+        parser.add_argument(
+            '--crxcavator-api-base-uri',
```

Comment:
update documentation to reflect that cli parameters are available

Reply:
@tayasteere Can you please update the docs to reflect the new CLI params?